solr/solrj/src/java/org/apache/solr/common/cloud/DocCollection.java

```diff
@@ -165,6 +205,8 @@ public static Object verifyProp(Map<String, Object> props, String propName, Obje
       case AUTO_ADD_REPLICAS:
       case READ_ONLY:
         return Boolean.parseBoolean(o.toString());
+      case PER_REPLICA_STATE:
```

Comment:
Can add this case to the group of two just above since they do the same thing already.